### DISCUSSION TITLE
Backup limit

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#894] [Ideas] [answered]
Ability to set a number of backups to keep.

**Comment by szaimen** [comment]
Why a specific number? (A specific number is btw not possible with borg afaik.)

**Reply by ghost** [reply]
Doesn't have to be a specific number can also be a check mark with keep 14 backups or similar.

**Comment by szaimen** [comment, marked as answer]
A retention policy is already in place which works not based on a number. See https://github.com/nextcloud/all-in-one/blob/3de963c238b77c1ba93ac92e614d836b8e046b48/Containers/borgbackup/backupscript.sh#L143 and https://borgbackup.readthedocs.io/en/stable/usage/prune.html

**Reply by ghost** [reply]
I think that would be a feature that is still missing in the AIO interface. This function would make https://github.com/nextcloud/all-in-one/issues/278 unnecessary. @szaimen what do you think of this function? I think this is a must have for any backup solution and many CPs also offer this (Plesk, ...). Could be applied what do you think?
